backend/cognito/template.yml

```
@@ -616,10 +621,10 @@ Outputs:
     Description: User Pool generated for this stack
     Value: !Ref UserPool
     Export:
-      Name: CognitoUserPool
+      Name: !If [ UseExportPrefix, !Sub "${ExportPrefix}-CognitoUserPool", "CognitoUserPool" ]
```

```suggestion
      Name: !Sub ${ExportNamePrefix}-CognitoUserPool"
```
And the same for all the exports